config/locales/manage_assessments.en.yml

```diff
@@ -60,6 +60,6 @@ en:
         description: Description
         heading: Outcomes for %{name}
         label: Label
-    outcome_coverages:
-      index:
+      show:
         add_a_class: Add a class
+        course_number: Course %{number}
```

Comment:
I like this. It explains why you were able to do this `<div><%= t(".course_number", number: @course.number) %></div>` in the view.